flytekit/clis/sdk_in_container/register.py

```diff
@@ -1,33 +1,46 @@
 from __future__ import absolute_import
 
 import click
+import logging as _logging
```

Comment:
maybe revert the changes in this file with the exception of the debug logs as I don't think they are necessary (and somewhat divergent from keeping logic associated with the objects themselves).

Reply:
but i like this...

Reply:
reverted, saving the diff for the next pr.